manifests/w/WildfireGames/0AD/r25860-alpha/WildfireGames.0AD.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WildfireGames.0AD
+PackageVersion: r25860-alpha
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://releases.wildfiregames.com/0ad-0.0.25b-alpha-win32.exe
```

Comment:
I've created a ticket at https://trac.wildfiregames.com/ticket/6388#comment:description to replace that with an MSIX package.